rust/frontend/src/api.rs

```diff
@@ -0,0 +1,55 @@
+use chroma_config::Configurable;
+use chroma_error::ChromaError;
+use chroma_sysdb::sysdb;
+use chroma_types::CreateDatabaseResponse;
+
+use crate::config::FrontEndConfig;
+
+#[async_trait::async_trait]
+pub trait ServerApi {
```

Comment:
Should this be outside of frontend? As bindings also wants to use it. I could go either way now that im writing this.